client/src/pages/SymptomsCheck.js

```diff
@@ -53,6 +59,12 @@ const Step1 = (props) => {
 
   return (
     <WizardStep>
+      <p className="mb-99">
+        <Link to="medical">
+          {/* By clicking on “X”, users can exit the symptoms questions to medical page */}
+          X
```

Comment:
probably better to use svg icon (we should use material design, still has to be added I think it's a work in progress)

Reply:
@robinv85 Thanks for the review and suggestion, i'll do that 👍